utils/build-tooling-libs

```diff
@@ -172,10 +173,14 @@ class Builder(object):
             "-DSWIFT_HOST_VARIANT_SDK=" + host_sdk,
             "-DSWIFT_HOST_VARIANT_ARCH=" + self.arch,
             "-DCMAKE_Swift_COMPILER_TARGET=" + host_triple,
-            "-DCMAKE_Swift_COMPILER=" + self.toolchain.swiftc,
+            "-DCMAKE_Swift_COMPILER=" + self.swiftc,
             "-DCMAKE_C_FLAGS=" + llvm_c_flags,
             "-DCMAKE_CXX_FLAGS=" + llvm_c_flags,
         ]
+        if self.swiftc:
+            cmake_args += [
+                f"-DCMAKE_Swift_COMPILER={self.swiftc}"
+            ]
```

Comment:
Hmm I think this is not correct. I think you meant default to use `self.toolchain.swiftc` but override it with `self.swiftc`. But the actual code doesn't use `self.toolchain.swiftc`?

Reply:
Oh, yes. That’s an artifact from an old version that I should have deleted.